content/en/docs/concepts/services-networking/dns-pod-service.md

```diff
@@ -150,6 +150,15 @@ pointing to the Pod's IP address. Also, Pod needs to become ready in order to ha
 record unless `publishNotReadyAddresses=True` is set on the Service.
 {{< /note >}}
 
+### Pod's hostnameFQDN field {#pod-hostnamefqdn-field}
+
+Currently, when a pod is configured to have fully qualified domain name (FQDN), the hostname field
```

Comment:
Hi. Watch out for “currently”, _especially_ when documenting behavior in an area where changes are planned. Consider rewording.

Also bear in mind https://kubernetes.io/docs/contribute/style/style-guide/#avoid-statements-about-the-future - as much as you can, document what is already merged rather than what is planned to come.